ghost/admin/app/components/gh-post-settings-menu.js

```diff
@@ -621,6 +621,14 @@ export default class GhPostSettingsMenu extends Component {
         });
     }
 
+    @action
+    saveTags() {
+        this.saveTagsTask.perform().catch((error) => {
+            this.showError(error);
+            this.post.rollbackAttributes();
```

Comment:
I wonder if we should skip that call - `tags` is a relationship rather than an attribute and doesn't get rolled back with `rollbackAttributes()` 🤔

Reply:
Ah okay, yeah I'll remove that

Reply:
Done!